main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	// "db"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/gin-gonic/gin"
+)
+
+func main() {
+	router := gin.Default()
+	fmt.Println("Hello World!")
+
+	router.Use(WelcomeEndpoint)
```

Comment:
`Print API path and current time on all API requests`
The first requirement is fulfilled.